packages/volto/src/components/manage/Contents/ContentsItem.jsx

```diff
@@ -97,6 +103,8 @@ export const ContentsItemComponent = ({
   order,
 }) => {
   const intl = useIntl();
+  const previewImageUrl = `${item['@id']}/${item.image_scales?.image?.[0]?.scales?.teaser?.download}`;
```

Comment:
@iRohitSingh this lookup should be added in a new component that can be placed in this file
which is called if the item type is image. Right now you would attempt to check this for everything
even if there is no need for it.

Reply:
done.